### DISCUSSION TITLE
Play from here on playlists and albums

### DISCUSSION THREAD

**Discussion by marcelveldt** [music-assistant/support#513] [Feature requests and ideas]
You can play an album --> the entire album (all tracks under it) is inserted and then played on the queue
You can play a playlist --> the entire playlist (all tracks under it) is inserted and then played on the queue

If you're browsing an album or a playlist and you click one of the tracks, the current default behavior is to play THAT track.
It will be inserted at the top of the queue and played. The existing items on the queue continue after that.

But what if you want to start playing an album/playlist from the track you selected. More like the behavior the Spotify app has.
There should be another enqueue option "Play from here", available when you're browsing an album or playlist listing.
If you select that, the entire playlist/album will be inserted in the queue from the position of the track you selected.

**Comment by OzGav** [comment]
The feature request has been implemented successfully. As such this feature request will be closed and locked. If there are enhancements desired then open a new [feature request](https://github.com/orgs/music-assistant/discussions/categories/feature-requests-and-ideas). If there is a question about how this works then check the [documentation](https://beta.music-assistant.io/), or ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord. If there are problems then ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord or open an [issue](https://github.com/music-assistant/support/issues). Thanks for your interest in seeing this idea being brought to life.
